scratch/logging-ns3-script.cc

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+
+#include "ns3/core-module.h"
+
+using namespace ns3;
+
+NS_LOG_COMPONENT_DEFINE ("Logger");
+
+int
+main (int argc, char *argv[])
+{
+
+  Simulator::Run ();
+  NS_LOG_DEBUG("Debug");
+  NS_LOG_INFO("Info");
+  NS_LOG_WARN("Warn");
+  NS_LOG_ERROR("Error");
+  NS_LOG_LOGIC("Logic");
+  // NS_LOG_FUNCTION("Function");
```

Comment:
Any particular reason why this is commented?

Reply:
The only reason, I commented this was because `NS_LOG_FUNCTION` has a little different format of logs. But, I already wrote a function to parse logs (for phase 2) taking this into consideration. So, I can uncomment this now.